index.html

```diff
@@ -317,7 +322,7 @@ <h2>Glossary</h2>
 
 <p><dfn data-lt="metadata|metadata" id="def_metadata" class="lint-ignore">Metadata</dfn> is additional information about data. Key types of metadata for <a class="termref" href="#internationalization">internationalization</a> are <a class="termref" href="#def_language_metadata">language metadata</a> and metadata to support <a class="termref" href="#def_bidirectional_text">bidirectional text</a>. Metadata has a scope, e.g., a string or a set of strings. In absence of explicit metadata, defaults might apply, e.g. defaults for the <a class="termref" href="#def_base_direction">base direction</a> of a text.</p>
 
-
+<p><dfn id="def_mojibake" class="lint-ignore">Mojibake</dfn> (文字化け). Garbled or incorrectly rendered or processed characters, generally caused by using the wrong <a>character encoding</a> to interpret the <a>code units</a> in a string or file. The word is Japanese in origin and is pronounced approximately <q>moh-gee-bah-kay</q>. For example, the word <q>文字化け</q> encoded as UTF-8 might be displayed as <q>&#x00e6;&#x0096;&#x0087;&#x00e5;&#x00ad;&#x0097;&#x00e5;&#x008c;&#x0096;&#x00e3;&#x0081;&#x0091;</q> if viewed using the (incorrect) character encoding <code>ISO-8859-1</code>.</p>
```

Comment:
If we're going to provide pronunciations, can we please use IPA.  I can think of a number of ways people could pronounce moh-gee-bah-kay, and if i didn't know better that would actually suggest the wrong encoding to me.  IPA is /mo.d͡ʒi.ba.ke/.

> if viewed using the (incorrect) character encoding

I suggest: if viewed in an application that thinks (incorrectly) that the character encoding is

Reply:
Happy to fix. Knew you'd provide this comment :-D.